src/main/kotlin/com/exactpro/th2/infraoperator/util/RabbitMQUtils.kt

```diff
@@ -76,8 +78,15 @@ internal fun KubernetesClient.collectRabbitMQRubbish(
         .forEach(exchanges::add)
 
     K_LOGGER.debug { "Actual set in RabbitMQ, queues: $queues, exchanges: $exchanges" }
+}
 
-    if (isHolderEmpty() || namespaces.isEmpty()) {
+internal fun ResourceHolder.filterRubbishResources(
+    client: KubernetesClient,
+    namespacePrefixes: Set<String>,
+    topicExchange: String,
+): ResourceHolder = this.apply {
```

Comment:
note: `this` qualifier is redundant here
```suggestion
): ResourceHolder = apply {
```